app/views/static/references.html.erb

```diff
@@ -9,30 +9,81 @@
+
+        <p class='static-page__text static-page__text--sinai'>Greek New Finds = Nikolopoulos, P. G. &Tau;&alpha; &nu;&#941;&alpha; &epsilon;&upsilon;&rho;&#942;&mu;&alpha;&tau;&alpha; &tau;&omicron;&upsilon; &Sigma;&iota;&nu;&#940; (Athens, 1998). English translation: A. Hatzopoulos, The New Finds of Sinai (Athens, 1999). [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/new-finds-of-sinai/oclc/913489510%26referer%3Dbrief_results&amp;sa=D&amp;source=editors&amp;ust=1683922505981979&amp;usg=AOvVaw2gGrj5nM76A_YNI2kLJPtF"', class: 'si-link' %>]</p>
+
+        <p class='static-page__text static-page__text--sinai'>Kamil = K&#257;mil, Mur&#257;d. <i>Catalogue of All Manuscripts in the Monastery of St. Catherine on Mount Sinai</i>. Harrassowitz, 1970. [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/catalogue-of-all-manuscripts-in-the-monastery-of-st-catharine-on-mount-sinai/oclc/859613867?referer%3Dbr%26ht%3Dedition&amp;sa=D&amp;source=editors&amp;ust=1683922505982771&amp;usg=AOvVaw2eubcGMhjKIcgPk9rRia8C"', class: 'si-link' %>] [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/42432167&amp;sa=D&amp;source=editors&amp;ust=1683922505983084&amp;usg=AOvVaw38nfNYq_H1cjtKjVMq3sbv"', class: 'si-link' %>]</p>
+
+        <p class='static-page__text static-page__text--sinai'>Lewis = Lewis, Agnes Smith. <i>Catalogue of the Syriac Manuscripts in the Convent of Saint Catherine on Mount Sinai</i>. Cambridge University Press, 1894. [<%= link_to 'Internet Archive', 'https://archive.org/details/catalogueofsyria00sainrich&amp;sa=D&amp;source=editors&amp;ust=1683922505983791&amp;usg=AOvVaw1Q37h4ADZ3K3zCnnsEG7Ij"', class: 'si-link' %>]</p>
+
+        <p class='static-page__text static-page__text--sinai'>LOC= <i>Manuscripts in St. Catherine&#39;s Monastery, Mount Sinai</i>. Library of Congress. [<%= link_to 'Library of Congress', 'https://www.loc.gov/collections/manuscripts-in-st-catherines-monastery-mount-sinai&amp;sa=D&amp;source=editors&amp;ust=1683922505984517&amp;usg=AOvVaw19FeW8w9OzVFkzT3mhQFdm"', class: 'si-link' %>]</p>
+
+        <p class='static-page__text static-page__text--sinai'>Meimaris = Meimaris, Yiannis E. <i>Katalogos ton neon aravikon cheirographon tes Hieras Mones Hagias Aikaterines tou Orous Sina = Kataluy al-Majtutat al-&rsquo;arabiyya al-muktasafa haditan bi-Dayr Sant Katarin al-muqaddas</i>. Ethnikon Hidryma Ereunon, 1985. [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/katalogos-ton-neon-aravikon-cheirographon-tes-hieras-mones-hagias-aikaterines-tou-orous-sina-kataluy-al-majtutat-al-arabiyya-al-muktasafa-haditan-bi-dayr-sant-katarin-al-muqaddas/oclc/1123922473?referer%3Dbr%26ht%3Dedition&amp;sa=D&amp;source=editors&amp;ust=1683922505985361&amp;usg=AOvVaw0xshNif-zj3YjExAa21Nsr"', class: 'si-link' %>]</p>
```

Comment:
URL should be: https://www.worldcat.org/title/katalogos-ton-neon-aravikon-cheirographon-tes-hieras-mones-hagias-aikaterines-tou-orous-sina-kataluy-al-majtutat-al-arabiyya-al-muktasafa-haditan-bi-dayr-sant-katarin-al-muqaddas/oclc/1123922473